todo_app/app.py

```diff
@@ -126,6 +151,8 @@ def donedoingitem(id):
     def newitems():
         itemname = request.form.get('Title')
         mongo.add_item_mongo(itemname)
+        app.logger.debug("user id %s created new item", flask_login.current_user.get_id())
+        app.logger.info("todo item %s was created", itemname)
```

Comment:
When logging things like this you should probably include the item id (but you'll either need to get mongo.add_item_mongo to return the created item's id, or move this log into the `mongo` module